tests/runner.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
-
+// process.on('unhandledRejection', e => {
```

Comment:
can likely remove, assuming older node versions we support handle ` --unhandled-rejections=strict`